mmv1/products/compute/RegionResizeRequest.yaml

```diff
@@ -0,0 +1,413 @@
+            type: String
+            description: |
+              The Compute Engine quota metric name.
+            output: true
+          - name: 'limitName'
+            type: String
+            description: |
+              The name of the quota limit.
+            output: true
+          - name: 'dimensions'
+            type: KeyValuePairs
+            description: |
+              The map holding related quota dimensions
+            output: true
+          - name: 'limit'
+            type: Integer
+            description: |
+              Current effective quota limit. The limit's unit depends on the quota type or metric.
+            output: true
+          - name: 'futureLimit'
+            type: Integer
+            description: |
+              Future quota limit being rolled out. The limit's unit depends on the quota type or metric.
+            output: true
+          - name: 'rolloutStatus'
+            type: String
+            description: |
+              Rollout status of the future quota limit.
+            output: true
+        - name: 'help'
+          type: NestedObject
+          description: |
+            [Output Only]
+          output: true
+          properties:
+          - name: 'links'
+            type: NestedObject
+            description: |
+              [Output Only]
+            output: true
+            properties:
+            - name: 'description'
+              type: String
+              description: |
+                Describes what the link offers.
+              output: true
+            - name: 'url'
+              type: String
+              description: |
+                The URL of the link.
+              output: true
+        - name: 'localizedMessage'
+          type: NestedObject
+          description: |
+            [Output Only]
+          output: true
+          properties:
+          - name: 'locale'
+            type: String
+            description: |
+              The locale used following the specification defined at https://www.rfc-editor.org/rfc/bcp/bcp47.txt. Examples are: "en-US", "fr-CH", "es-MX"
+            output: true
+          - name: 'message'
+            type: String
+            description: |
+              The localized error message in the above locale.
+            output: true
+  # Status.lastAttempt
+  - name: 'lastAttempt'
+    type: NestedObject
+    description: |
+      [Output only] Information about the last attempt to fulfill the request. The value is temporary since the ResizeRequest can retry, as long as it's still active and the last attempt value can either be cleared or replaced with a different error. Since ResizeRequest retries infrequently, the value may be stale and no longer show an active problem. The value is cleared when ResizeRequest transitions to the final state (becomes inactive). If the final state is FAILED the error describing it will be storred in the "error" field only.
+    output: true
+    properties:
+    - name: 'error'
```

Comment:
is this part of the "errors" structure?
Both seem to have similar fields, e.g. locale and locialised message...

Reply:
Yes, both refers to errors from [mixer_operations.proto](https://source.corp.google.com/piper///depot/google3/cloud/cluster/api/mixer_operations.proto;rcl=684803196;l=53)

Reply:
Is it possible to have a structure we would use in both the places not to copy-paste the same fields so many times?
@rileykarson , maybe you know...

Reply:
We do not allow shared types in our codebase- we used to have them, and they introduce a ton of problems when there is variance between the occurrences. There is _very often_ drift between initially-shared types over O(mopnths-years) and we had recurring issues so we removed the ability to specify them to resolve the issues structurally.

I'll preemptively ack that that's very unlikely to be a problem in this specific case and there are some cases where it would be valid and continue to be valid- we saw numerous cases where we were assured they would always be the same, something changed, and suddenly there was drift / variance than invalidated sharing the type in the first place.